Doc/whatsnew/2.7.rst

```diff
@@ -130,7 +130,6 @@ For projects wishing to migrate from Python 2 to Python 3, or for library
 and framework developers wishing to support users on both Python 2 and
 Python 3, there are a variety of tools and guides available to help decide
 on a suitable approach and manage some of the technical details involved.
-The recommended starting point is the :ref:`pyporting-howto` HOWTO guide.
```

Comment:
It's odd to see an update to what's essentially a historic document. Possibly the sentence could remain but the link removed? That's also weird, but preserves a little more of the history.